Appsilon/rhino
# Include shinyApp string in app.R so it is recognized as Shiny app

### Changes
Closes #204, closes #216.

According to this [article](https://rtask.thinkr.fr/shinyapp-runapp-shinyappdir-difference/), having `shiny::runApp()` in `app.R` won't work in deployment, so I didn't even test it. I did learn however, that just having `shinyapp` string anywhere in `app.R` is sufficient to make this file recognizable as a Shiny app.

### How to test
Initialize a project and open `app.R` in RStudio. You should see "Run app" and "Publish Application" buttons; they should work.
